src/components/AssetHeader/AssetActions.tsx

```diff
@@ -25,6 +29,15 @@ export const AssetActions: React.FC<AssetActionProps> = ({ assetId, accountId, c
   } = useWallet()
   const asset = useAppSelector(state => selectAssetById(state, assetId))
 
+  useEffect(() => {
+    try {
+      chainAdapterManager.byChainId(asset.chainId)
```

Comment:
`byChainId` returns a `promise` - are we missing an `await`?

Reply:
actually we should change that. `ChainAdapter.getChainID()` is no longer an async function. but we'll need to change that in ChainAdapters first. for now, we need to await that

Reply:
See https://github.com/shapeshift/lib/pull/668

Reply:
I imagine https://github.com/shapeshift/lib/pull/668 will go in pretty quick, so @gomesalexandre it's probably fine to leave this how it is and update this PR to use the latest `chain-adapters` package before we merge it.

Reply:
Nice one! Moving to draft until #668 is merged, will bump deps and reopen

Reply:
gomie @gomesalexandre https://github.com/shapeshift/lib/pull/668 is merged, let's go

Reply:
Bumped, still unable to test the fix locally once again though as we're still rugged by Imperator.co